### ISSUE TITLE
Helm V1 server is operating server build

### ISSUE THREAD

**Issue by RowdyMuncher** [HelmSecure/armbian-images#32]
Good morning all of us with Helm server v1 are still waiting patiently for our software so we can not only use our heavy desk weights as it was intended but also have access to our data. Can you please let use know when we can down load the server software for V1 please.

**Comment by jonto** [comment]
I’m waiting for v1 images as well. Would love an update to learn if this is being worked on.

**Comment by steakscience** [comment]
It's been too long and too quiet

I think it won't ever come out, there's no financial incentive to do so. The only solution left is for the community to reverse engineer a firmware

**Comment by TextBandit** [comment]
Just upvoting the need for a solution for v1 Helms -- it would be nice to get some answer here, or better, a solution for those of us with the triangles.

**Comment by tmiller101** [comment]
Agreed.  Would like to be able to do something with this device, besides fil a landfill..

**Comment by ukd1** [comment]
@dsigurds also hoping this could happen still?

**Comment by justinhamlett** [comment]
I wonder if there is enough community interested in getting the V1 operational with Armbian or anything to use these devices again. Sadly, the promised V1 update does not seem to be coming. 

Could someone from the old team put us in the direction of getting the V1 firmware updated to use.

**Comment by bryce-shashinka** [comment]
I have given up and scrapped mine. I stopped holding my breath.
